common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -890,10 +894,12 @@ class EnrollmentEmbargoTest(EnrollmentTestMixin, UrlResetMixin, ModuleStoreTestC
     EMAIL = "bob@example.com"
     PASSWORD = "edx"
 
+    URLCONF_MODULES = ['embargo']
```

Comment:
This was part of a cleanup to stop passing arguments into `setUp`, because it makes creating `TestCase` mixins difficult.